apps/src/blockly/themes/cdoBlockStyles.mjs

```diff
@@ -0,0 +1,79 @@
+// Whenever updating the styles in this file, be sure to also update the styles
+// in cdoAccessible.js as well. The accessible styles can be generated using
+// accessibleColors.mjs.
+// From this directory in your terminal, enter:
```

Comment:
Is there a reason you have these instructions in both files?

Reply:
If someone is updating the default styles, I want them to know to update the accessible ones too. If someone happens to be checking out the accessible styles only, I want them to understand where they should come from.

Reply:
That makes sense  - thanks!